src/LuaTransformer.ts

```diff
@@ -3742,7 +3742,8 @@ export class LuaTransformer {
         }
         return filePath
             .replace(/\.\//g, "")
-            .replace(/\//g, ".");
+            .replace(/\//g, ".")
+            .replace(/\.json$/, "");
```

Comment:
Could you do the json replacement fist? Otherwise it would break modules named `json`, see #418.